test/spec/page.spec.js

```diff
@@ -31,4 +31,23 @@ describe( 'Pages mode', () => {
 
     } );
 
+    describe( 'Flip to page on multipages form', () => {
+
+        it( 'flip to second page', () => {
+            const form = loadForm( 'pages-comment.xml' );
+            form.init();
+
+            const currentPage = form.pages.current;
+            const pageTwoTextAreaHiddenComment = form.view.html.querySelector( '[data-for="/data/item2"]' );
+            const pageComment = form.view.html.querySelector( 'input[name="/data/item2"]' ).closest( '[role="page"]' );
+            const pageTwoTextAreaCommentAncestor = pageTwoTextAreaHiddenComment.closest( '[role="comment"]' );
+    
+            form.pages.flipToPageContaining( [ pageTwoTextAreaCommentAncestor ] );
+            expect( currentPage ).not.toEqual( form.pages.current );
+            expect( pageComment ).toEqual( form.pages.current );
```

Comment:
Travis testing fails because in the latest master we moved away from Jasmine to Chai (so `not.to.equal` and `to.equal`, I think)

Reply:
merged the latest master for testing the changes and code updated